sopel/modules/wikipedia.py

```diff
@@ -105,7 +101,8 @@ def mw_snippet(server, query):
     return snippet['extract']
 
 
-@rule(r'.*\/([a-z]+\.wikipedia\.org)\/wiki\/((?!File\:)[^ ]+).*')
+page_url = r'/([a-z]+\.wikipedia\.org)/wiki/((?!File\:)[^ ]+)'
+@url(page_url)
 def mw_info(bot, trigger, found_match=None):
```

Comment:
The `match` parameter should be always provided when `@url` is used instead of `@rule`.

```suggestion
def mw_info(bot, trigger, found_match):
```